easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb

```diff
@@ -0,0 +1,28 @@
+
+name = 'GenotypeHarmonizer'
+version = '1.4.14'
+
+homepage = 'https://github.com/molgenis/systemsgenetics/wiki/Genotype-Harmonizer'
+description = """The Genotype Harmonizer is an easy to use command-line tool that allows harmonization of genotype data stored using different file formats with different and potentially unknown strands."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+easyblock = 'PackedBinary'
+
+dependencies = [('Java', '1.7.0_80')]
+
+#
+# Example URL
+# http://www.molgenis.org/downloads/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-dist.tar.gz 
+#
```

Comment:
please remove this comment block, these get outdated too easily (e.g. when `--try-software-version` is used)